### DISCUSSION TITLE
Submenu too long for mobile

### DISCUSSION THREAD

**Discussion by rbiddis1** [daattali/beautiful-jekyll#1319] [General]
I'm trying to find a way to get my long submenu to scroll on a mobile device since it doesn't all fit on the screen?

I tried in CSS like this:

.dropdown-menu {
  max-height: 480px;
  overflow-y: scroll;
}

But I don't like the appearance. Any other suggestions?

Here's my site for reference. https://bettercomputertech.com

**Comment by daattali** [comment]
That's not a bad solution honestly. But I think you may want to just rethink your menu entirely, because having that many submenus is arguably too many. At some point, it makes more sense to make a page to list them rather than have such a long list in a menu. Even on my laptop I can't see the entire list. 

(PS. at the bottom, where it says PRIVACY DISCLAIMER, I would add some more space or perhaps a bullet point (`•`) because it looks like the two words are together when they are in fact two separate links)

**Reply by rbiddis1** [reply]
Those are great ideas. Thanks for your feedback. I was actually considering the single page idea.

Thanks for your time and a great theme.
